### PR TITLE
Added add/remove myself from list of speakers button to motion detail…

### LINKED ISSUE #1851: Feature: Add me to the Speachlist Button at Motions
Offers a parallel reading and writing of personal notes, with the option to participate to the discussion.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
… view

Closes #1851

### CODE REVIEW

**Review by bastianjoel — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
Works, but still needs discussion with ui experts...

**Comment by emanuelschuetze** [comment]
I close this PR. We want to redesign the motion detail toolbar to get more context action buttons (like project, add to list of speakers, pdf) directly in motion detail - without 3-dot-menu. We will create mockups and new issue later in 4.2

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts [hunk 1]
```diff
@@ -207,7 +207,7 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
         if (speakerToDelete && (await this.promptService.open(title))) {
             await this.speakerRepo.delete(speakerToDelete.id);
             this.filterNonAvailableUsers();
-            this.interactionService.kickUsers([speaker.user], `Removed from the list of speakers`);
+            this.interactionService.kickUsers([speakerToDelete.user], `Removed from the list of speakers`);
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:210]: This looks like a bug which should be fixed regardless, right?

Comment (luisa-beerboom) [reply client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:210]: Absolutely, I'll make an issue

Comment (luisa-beerboom) [reply client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:210]: -> #2232

**Pull request closed without merging**